src/components/ViewResponses/Responses.js

```diff
@@ -0,0 +1,43 @@
+import { Card, Typography } from "antd";
+
+const Response = (props) => {
+  let { allResponses, userInfo } = props;
+
+  const { Title } = Typography;
+  allResponses && allResponses.length !== 0 && (
+    <Title level={3}> Responses</Title>
+  );
+  allResponses.map((response, index) => {
+    let questions = JSON.parse(response.plaintext);
+    console.log("r", response);
+    return (
+      <>
+        <Card
+          title={
+            userInfo[response.pubkey]?.name ||
+            "Anonymous Response " + (index + 1)
+          }
+        >
+          {questions.map((question) => {
+            console.log(question);
+            return (
+              <Card type="inner" title={question.question}>
+                <div style={{ display: "flex", flexDirection: "column" }}>
```

Comment:
instead of inline styling can use styled-components/emotion. Will make styling the components much more manageable

Reply:
yes that is the plan, we can pick it up after moving to typescript